docker-entrypoint.sh

```diff
@@ -116,6 +116,10 @@ if [[ "$RECORDLOG_ENABLED" ]]; then
     JQ_FILTERS_CONFIG="$JQ_FILTERS_CONFIG | .recordLog.enabled=true"
 fi
 
+if [[ "$STORAGE_QUOTA_ENABLED" ]]; then
```

Comment:
Why not set `utapi.metrics` with the metrics array value as `['location']`? That way we can extend it later for the other metric granularities.

Reply:
Seems like we may eventually want to introduce other metrics.